vql/linux/audit.go

```diff
@@ -501,6 +521,26 @@ func (self *AuditWatcherService) startEventQueue() {
 	}
 
 	atomic.AddInt64(&self.currentMessagesQueuedCounter, -count)
+	elapsed := time.Now().Sub(then)
+	self.logger.Debug("drained %v events from queue in %v", count, elapsed)
+}
+
+// Ensure that no events sit in the queue for more than 500ms
+func (self *AuditWatcherService) startEventQueueMaintainer() {
+	defer self.wgDec("startEventQueueMaintainer")
+
+	timeout := time.NewTicker(500 * time.Millisecond)
+	defer timeout.Stop()
```

Comment:
Does this have any effect? You create the ticker but I don't see where you read from it? In the loop below you read from queueFlushTimer ?

Reply:
Thanks, yep. That's a leftover from before I had factored out the flushing and the timer. It should be removed.